consoles/VNC.pm

```diff
@@ -595,46 +595,50 @@ sub init_x11_keymap {
     my ($self) = @_;
 
     return if $self->keymap;
-    $self->keymap($keymap_x11);
+    # create a deep copy - we want to reuse it in other instances
+    my %keymap = %$keymap_x11;
```

Comment:
Isn't this only regular `HASH` copy? By deep copy I usually understand copying `HASH`es, `ARRAY`s, etc.. nested in other `HASH` and so on which usually needs some more perl magic. I know that in this case it is the same as %keymap is flat. Just for future reference and if there is some neat trick I was ignoring.